quinn-h3/src/connection.rs

```diff
@@ -103,6 +103,7 @@ pub(crate) struct ConnectionInner {
 
 impl ConnectionInner {
     fn drive(&mut self, cx: &mut Context) -> Result<bool, DriverError> {
+        self.poll_recv_control(cx)?;
         self.poll_incoming_uni(cx)?;
         self.poll_send(cx)?;
         self.poll_recv_control(cx)?;
```

Comment:
Should we poll this twice?

Reply:
Got it better in a separate commit. Polling control upon opening. It's important to use the settings (first frame on control stream) as soon as possible as it contains compression enablement parameters.